src/include/block_def.h

```diff
@@ -213,6 +213,13 @@ struct __wt_bm {
     bool is_live; /* The live system */
 };
 
+typedef enum {
```

Comment:
Enums that are nested inside structs need to have their name qualified like: `__wt_block::WT_CKPT_NONE`.
The path of least resistance is just to move them out of the struct definition so I've done that.